site/specs/numbers.yml

```diff
@@ -28898,17 +28898,17 @@ paths:
                              <Name>Sample Report 1</Name>
                              <Id>100020</Id>
                              <Description>Sample Report 1 Description</Description>
-                          </Report>
+                             </Report>
```

Comment:
@xxlaykxx We've seen this in other PRs, but if you could maybe spend a minute to see if you can figure out why our XML parser messed up the indentation in this example, that would be great